<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_CadToolBar_FileOpt</class>
 <widget class="QWidget" name="QG_CadToolBar_FileOpt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>96</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QG_CadToolBar_FileOpt</string>
  </property>
  <widget class="QToolButton" name="btnNew">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>29</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../res/extui/extui.qrc">
     <normaloff>:/extui/new.png</normaloff>
     <normalon>:/extui/new_ON.png</normalon>
     <disabledoff>:/extui/new.png</disabledoff>
     <disabledon>:/extui/new_ON.png</disabledon>
     <activeoff>:/extui/new.png</activeoff>
     <activeon>:/extui/new_ON.png</activeon>
     <selectedoff>:/extui/new.png</selectedoff>
     <selectedon>:/extui/new_ON.png</selectedon>:/extui/new.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>29</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../res/extui/extui.qrc">
     <normaloff>:/extui/fileopen.png</normaloff>
     <normalon>:/extui/fileopen_ON.png</normalon>
     <disabledoff>:/extui/fileopen.png</disabledoff>
     <disabledon>:/extui/fileopen_ON.png</disabledon>
     <activeoff>:/extui/fileopen.png</activeoff>
     <activeon>:/extui/fileopen_ON.png</activeon>
     <selectedoff>:/extui/fileopen.png</selectedoff>
     <selectedon>:/extui/fileopen_ON.png</selectedon>:/extui/fileopen.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>60</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../res/extui/extui.qrc">
     <normaloff>:/extui/save.png</normaloff>
     <normalon>:/extui/save_ON.png</normalon>
     <disabledoff>:/extui/save.png</disabledoff>
     <disabledon>:/extui/save_ON.png</disabledon>
     <activeoff>:/extui/save.png</activeoff>
     <activeon>:/extui/save_ON.png</activeon>
     <selectedoff>:/extui/save.png</selectedoff>
     <selectedon>:/extui/save_ON.png</selectedon>:/extui/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnSaveAs">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>60</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolTip">
    <string>另存为DXF</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../res/extui/extui.qrc">
     <normaloff>:/extui/saveas.png</normaloff>
     <normalon>:/extui/saveas_ON.png</normalon>
     <disabledoff>:/extui/saveas.png</disabledoff>
     <disabledon>:/extui/saveas_ON.png</disabledon>
     <activeoff>:/extui/saveas.png</activeoff>
     <activeon>:/extui/saveas_ON.png</activeon>
     <selectedoff>:/extui/saveas.png</selectedoff>
     <selectedon>:/extui/saveas_ON.png</selectedon>:/extui/saveas.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>80</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="LeadPara">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>文件操作</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res/extui/extui.qrc"/>
 </resources>
 <connections/>
</ui>
